zipkin-junit/pom.xml

```diff
@@ -55,6 +55,14 @@
       <artifactId>junit</artifactId>
     </dependency>
 
+    <dependency>
+      <!-- okhttp uses jul -->
+      <groupId>org.slf4j</groupId>
```

Comment:
so do we (in the core jar), so .. we'll need this in the server. Also we need to make sure we didn't accidentally add a strict dep here on slf4j either as it is unnecessary.